addons/website_mass_mailing_sms/views/snippets/snippets_templates.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="s_newsletter_block_sms_template" groups="base.group_user">
```

Comment:
Don't we use no first indentation in new XML files ? (at least we do on our side... :grimacing:)